src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.UnitTests/Utilities/LanguageServerTestComposition.cs

```diff
@@ -25,6 +25,7 @@ internal sealed class LanguageServerTestComposition
             ExtensionAssemblyPaths: extensionPaths ?? [],
             DevKitDependencyPath: devKitDependencyPath,
             RazorDesignTimePath: null,
+            CSharpDesignTimePath: null,
```

Comment:
Should we just be creating a generic mechanism for pushing MSBuild properties rather than adding one each time we need this?

Reply:
Going to err on the side of caution and leave this for now. I think initialization is definitely ripe for review, once cohosting graduates and we remove rzls support. At the moment there are still two very different ways of working with Roslyn in this project which complicates things and makes changes potentially fragile.